src/bots/telegram/client.js

```diff
@@ -222,6 +224,23 @@ function prepareMessage(msg: Telegram$Message): string {
   return text;
 }
 
+function prepareQuote(quote: string): string {
+  if (!quoteLength) {
+    return quote;
+  }
+
+  // get first three lines
+  let prepared = quote.split('\n').slice(0, 3).join('\n');
```

Comment:
I suggest that we extract that `3` into a config parameter. Probably `"quoteLimits": { "characters": 140, "lines": 3 }`?